### ISSUE TITLE
Export named 'CDN' not found in module

### ISSUE THREAD

**Issue by JackoPlane** [Didas-git/lilybird#14]
After creating a simple demo using `$ bun create @lilybird`, I'm receiving a `CDN` module not found error. The default `src/index.ts` and `src/listeners/ready.ts` are used as provided by the `bun create` command.

# Configuration

### Environment

- `$ bun --version` `1.0.26`
- `$ uname -a` `Darwin <hostname> 23.4.0 Darwin Kernel Version 23.4.0: Fri Jan 12 22:40:34 PST 2024; root:xnu-10063.100.610.0.2~19/RELEASE_ARM64_T6000 arm64 arm Darwin`

### package.json
```
{
  "name": "Test",
  "description": "Test2",
  "version": "1.0.0",
  "type": "module",
  "main": "./src/index.ts",
  "scripts": {
    "start": "bun ."
  },
  "dependencies": {
    "@lilybird/handlers": "^0.3.3",
    "@lilybird/jsx": "^0.2.0",
    "lilybird": "^0.5.3"
  },
  "devDependencies": {
    "bun-types": "^1.0.30",
    "typescript": "^5.4.2"
  }
}
```

### tsconfig.json
```
{
    "compilerOptions": {
        "target": "ESNext",
        "module": "Node16",
        "moduleResolution": "Node16",
        "outDir": "dist",
        "baseUrl": ".",
        "strict": true,
        "types": [
            "./globals.d.ts",
            "bun-types"
        ]
    },
    "include": [
        "src/**/*"
    ]
}
```

# Steps to reproduce

1. Create new project, `$ bun create @lilybird`
2. What will the name of the directory be? `sample`
3. What package manager would you like to use? `Bun`
4. What language will you be using? `Typescript`
5. What typescript config style would you like to use? `Strict`
6. Choose what packages you would like to install `@lilybird/jsx` and `@lilybird/handlers`
7. Fill out your package.json, name: `sample`, description: `sample`, version: `1.0.0`
8. `$ cd sample`
9. `$ bun .`

# Expected results

Bun able to parse and execute code

# Actual results

```
- | (function (entry, fetcher)
SyntaxError: Export named 'CDN' not found in module '<$pwd>/node_modules/lilybird/dist/index.js'.
```

**Comment by JackoPlane** [comment]
If there is any additional information I can provide I'm happy to assist.

For what it's worth, Using Bun `v1.0.30` I am still experiencing this issue.

**Comment by Didas-git** [comment]
I apologise for this.
The latest `handlers` version is not compatible with lilybird 0.5.3, there was a mistake when publishing the alpha versions for handlers, this issue will be fixed once 0.6 becomes stable hopefully early next week.
For now i recommend downgrading the handlers to 0.3.1
